src/Constants.php

```diff
@@ -46,13 +67,20 @@ public static function init($dir)
 	private static function env_set($key, $default)
 	{
 		// Check if we have env value
-		$env_value = env($key, $default);
+		$env_value = env($key);
 
-		if (! isset($env_value)) {
-			return;
-		}
+        // If not value or default move along
+        if (!isset($env_value) && !isset($default)){
+            return;
+        }
+
+        // Use the default if env_value is not set
+        if(!isset($env_value)){
+            DotEnv::set($key,$default);
+            $env_value = $default;
+        }
 
-		// if not defined
+		// Make default env_vars global vars.
```

Comment:
Currently, vars defined in .env.php do not automatically become defined constants. However vars defined in $env_vars at the top of this file do. 

Conversely, vars defined in $env_vars (prior to this PR) were not available via DotEnv::get(), however vars defined in .env.php are.